dashboard/app/views/regional_partners/show.html.haml

```diff
@@ -84,6 +87,14 @@
 = link_to 'Back', regional_partners_path
 %br
 %h2 Regions
+%p
+  Region mapping for this partner can be replaced by uploading a CSV file that includes the header
```

Comment:
I prefer the CSV format explanation in your [spec](https://docs.google.com/document/d/16CgWiwymfQgcxxDVCAhTCX6INRGo6GfaYeA6tJfcSw8/edit#). It clearly shows the file should have 2 columns {PartnerId, Region}, and it has examples. Ideally, the explanation here is all users need to create a valid CSV file.

Reply:
I updated the requirements since writing the spec--since this only supports upload for a single partner, the csv can be:
Region
12345
WA
23456
...
It can also still be the {PartnerId, Region} format since it's based on a 'Region' header. I didn't want the explanation to take up too much space on the page since it won't be used very much. I could add the sample format but it would take quite a few lines if you think this explanation isn't clear enough

Reply:
You can add a few examples in one line. Preferably, there is no room for misinterpreting the instruction.

Reply:
added a small example